### DISCUSSION TITLE
Apple ideas

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#700] [Ideas]
Apples feel underused right now, but that might be just me. But here are a few ideas anyways

Applesauce and maybe Honey'd apples?

May add more if I think of some

**Comment by vectorwing** [comment]
Could you please slow down with all those new discussion threads? I understand your enthusiasm, but it's starting to become spammy.
I suggest compiling your ideas into a few posts (for example, meal suggestions).

**Reply by Aceplante** [reply]
Apologies, sure. I will do that later today. My thought process is that they are separate ideas people can vote on individually in case they like some parts but not others.
